ts/demo-functions/build/helm_template.Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM node:10-alpine as builder
+
+RUN mkdir -p /home/node/app && \
+    chown -R node:node /home/node/app
+
+USER node
+
+WORKDIR /home/node/app
+
+# Install dependencies and cache them.
+COPY --chown=node:node package*.json ./
+RUN npm ci
+
+# Build the source.
+COPY --chown=node:node tsconfig.json .
+COPY --chown=node:node src src
+RUN npm run build && \
+    npm prune --production && \
+    rm -r src tsconfig.json
+
+#############################################
+
+FROM node:10-alpine
+
+RUN apk add bash curl git
+RUN apk update
+
+ENV HELM_LATEST_VERSION="v3.2.1"
+RUN curl -fsSL -o /helm-${HELM_LATEST_VERSION}-linux-amd64.tar.gz https://get.helm.sh/helm-${HELM_LATEST_VERSION}-linux-amd64.tar.gz && \
```

Comment:
Download this and kpt CLI in the `builder` image and then copy them here.

Reply:
Done